__tests__/app.test.js

```diff
@@ -125,3 +125,58 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article/comments", () => {
+  test("200: sends all the comments of specific article", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toEqual([
```

Comment:
This test doesn't make your endpoint very extensible. You'd be better checking the length and then doing a forEach and using the .toMatchObject jest matcher to check the properties. That means any properties added in future won't break your tests.